app/models/cron_job_scheduler.rb

```diff
@@ -0,0 +1,13 @@
+class CronJobScheduler
+  def initialize
+    Dir.glob(glob).each { |f| require f }
+  end
+
+  def schedule
+    CronJob.subclasses.each { |job| job.schedule }
+  end
+
+  def glob
```

Comment:
It's good to limit the public interface of classes we create to only those methods we expect to get called publicly, so maybe this method should be `private`/